src/python/pants/pantsd/process_manager.py

```diff
@@ -401,6 +401,14 @@ def _kill(self, kill_sig):
     if self.pid:
       os.kill(self.pid, kill_sig)
 
+  def _noop_fork_context(self, func):
```

Comment:
When is this the correct thing to use?

Reply:
Inlined and moved this docstring into the `daemonize` pydoc.